### PR TITLE
tests(ipld): temporary skip a test not to see the race detector complaining

### PR DESCRIPTION
**Pull request by Wondertan**

### CODE REVIEW

File: ipld/read_test.go [hunk 1]
```diff
@@ -109,6 +109,8 @@ func TestBlockRecovery(t *testing.T) {
 }
 
 func TestRetrieveBlockData(t *testing.T) {
+	t.Skip("skipping as it spawns too many goroutines")
```
Comment (liamsi) [inline ipld/read_test.go:112]: Alright. It is not easily possible to only skip the large one, right?

Comment (Wondertan) [reply ipld/read_test.go:112]: Possible

Comment (liamsi) [reply ipld/read_test.go:112]: Let's do that then :)

Comment (Wondertan) [reply ipld/read_test.go]: Done

Comment (liamsi) [reply ipld/read_test.go]: Thanks for putting this in a separate PR btw 🙏🏼

File: ipld/read_test.go [hunk 2]
```diff
@@ -123,7 +123,9 @@ func TestRetrieveBlockData(t *testing.T) {
 		{"128x128(max)", MaxSquareSize},
 	}
 	for _, tc := range tests {
-
+		if tc.squareSize == MaxSquareSize {
+			t.Skip("skipping as it spawns too many goroutines")
+		}
```
Comment (liamsi) [inline ipld/read_test.go:128]: dopey-dopey

**Review by renaynay — APPROVED** [review]

**Review by liamsi — APPROVED** [review]
Thanks!

**Pull request merged**